### PR TITLE
Add support for multiple files

### PR DESCRIPTION
**Pull request by frclba**
Solving issue #3 - Support for a directory of files

### CODE REVIEW

**Comment by shiffman** [comment]
Thank you! Wonderful! Will it still work if you pass in a single file name? Also, is this compatible with the changes in #2? I think I should probably finalize and merge that one first?

**Comment by frclba** [comment]
Right now isn't working if we pass a single file name. I can fix that.
It's better to merge the first PR and then change on top of that.
Thanks ! :+1:

**Comment by shiffman** [comment]
Ready to take a look at this now that #4 is merged!

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,9 @@
 *.txt
 .vscode/
 vectors.json
+<<<<<<< HEAD
```
Comment (shiffman) [inline .gitignore]: Looks like some merge conflicts got left in here? Let me try to resolve and then can you double-check?

**Comment by frclba** [comment]
Now is working with either a single file or a folder with multiple files.

**Comment by shiffman** [comment]
Thank you!

**Pull request merged**